### PR TITLE
Fix minor mistakes throughout the documentation

### PR DESCRIPTION
**Pull request by MewPurPur**
* Uses of "instance" as a verb, begone.
* While at it... Uses of "For instance" replaced with "For example" throughout the documentation, as it's slightly shorter, more familiar, and doesn't hold a special meaning in Godot.
* Grammar etc.

### CODE REVIEW

File: doc/classes/Image.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 			<param index="1" name="src_rect" type="Rect2i" />
 			<param index="2" name="dst" type="Vector2i" />
 			<description>
-				Alpha-blends [param src_rect] from [param src] image to this image at coordinates [param dst], clipped accordingly to both image bounds. This image and [param src] image [b]must[/b] have the same format. [param src_rect] with not positive size is treated as empty.
+				Alpha-blends [param src_rect] from [param src] image to this image at coordinates [param dst], clipped accordingly to both image bounds. This image and [param src] image [b]must[/b] have the same format. [param src_rect] of non-positive size is treated as empty.
```
Comment (clayjohn) [inline doc/classes/Image.xml:29]:
```suggestion
				Alpha-blends [param src_rect] from [param src] image to this image at coordinates [param dst], clipped accordingly to both image bounds. This image and [param src] image [b]must[/b] have the same format. [param src_rect] with non-positive size is treated as empty.
```

This should be "with" not "of" same goes with the other lines in this file

File: doc/classes/MultiMeshInstance2D.xml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance2D" inherits="Node2D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances a [MultiMesh] in 2D.
+		Node that instantiates a [MultiMesh] in 2D.
```
Comment (clayjohn) [inline doc/classes/MultiMeshInstance2D.xml:4]: This should be "instances". In the Multimesh individual units are called "instances" and you don't instantiate them. Same goes with the change below

File: doc/classes/MultiMesh.xml [hunk 3]
```diff
@@ -100,7 +100,7 @@
 			By default, all instances are drawn but you can limit this with [member visible_instance_count].
 		</member>
 		<member name="mesh" type="Mesh" setter="set_mesh" getter="get_mesh">
-			[Mesh] resource to be instanced.
+			[Mesh] resource to be instantiated.
```
Comment (clayjohn) [inline doc/classes/MultiMesh.xml:103]: This should be reverted, "instances" is the correct term in a MultiMesh

File: doc/classes/MeshInstance3D.xml [hunk 4]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MeshInstance3D" inherits="GeometryInstance3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances meshes into a scenario.
+		Node that instantiates meshes into a scenario.
```
Comment (clayjohn) [inline doc/classes/MeshInstance3D.xml:4]: The changes in this file should be reverted, "instance" is the correct term here

File: doc/classes/MultiMeshInstance3D.xml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MultiMeshInstance3D" inherits="GeometryInstance3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances a [MultiMesh].
+		Node that instantiates a [MultiMesh].
```
Comment (clayjohn) [inline doc/classes/MultiMeshInstance3D.xml:4]: Changes in this file should be reverted as "instance" is the correct terminology here

**Review by clayjohn — COMMENTED** [review]
Nice work! I left a few comments making suggestions

**Review by mhilbrunner — APPROVED** [review]

**Pull request merged**